docker/README.md

```diff
@@ -0,0 +1,46 @@
+## docker installation
+
+1. build the tradenote image with your custom parameters
+```
+docker build . -t tradenote --build-arg PARSE_INIT=APPLICATION_ID --build-arg PARSE_URL=http://pi:1337/parse
+```
+
+2. run mongodb, parse server tradenote via docker compose file
+```
+docker-compose up -d
+```
+
+3. modify PARSE_INIT, PARSE_MASTER_KEY, PARSEHOST in addclass.sh to match your environment, run addclass.sh to add classes
+```
+chmod +x ./addclass.sh
+./addclass
+```
+
+4. register user via http://server:7777/register
+
+5. modify parameters to match your environment in localdashboard.yaml, run a standalone parse-dashboard container on local computer
```

Comment:
I personally run and use the dashboard (quite regularly). So in my opinion, I would not just make it a "local" dashboard. So
1- I would add the parse-dashboard to the docker-composer
2- Make step 8 optional
What do you think ?

Reply:
my thought is users like me dont need the dashboard, only the developer like you need it

why I made the "local" dashboard, because the modifyuser.sh is not working, users need this to add columns and data to _User class, once users added data, they can delete the local dashboard

another reason is parse-dashboard doesn't work on arm cpu, I cant make the whole tradenote services work on raspberry pi if add dashboard to the default docker-compose

but I am not sure, are users really dont need the dashboard? please let me know. this is my first time using parse:)